examples/vite-webpack-rspack/webpack/src/Image.tsx

```diff
@@ -1,9 +1,10 @@
-import React from 'react';
-import { useState } from 'react';
+import React, { useState } from 'react';
 import logo from './assets/webpack-logo.png';
 
-const WebpackImage = () => {
+const WebpackImage = ({ foo }: { foo: number }) => {
```

Comment:
missing types on the component in host

Reply:
I know its jsx but it can still use and should have the @mf-types/ folder extracted

Reply:
The foo: number is added for example

Reply:
Thanks 👍 it helps a lot